org.eclipse.scout.rt.client/src/main/java/org/eclipse/scout/rt/client/ui/basic/table/internal/InternalTableRow.java

```diff
@@ -528,6 +516,21 @@ private void setCellChanged(ICell cell, int changeBit) {
     return indexesByCell;
   }
 
+  private Integer setBit(Integer bitMask, int bitPos) {
+    if (bitMask == null) {
+      bitMask = 0;
+    }
+    bitMask |= 1 << bitPos;
+    return bitMask;
```

Comment:
This version saves a boxing and unboxing operation in case `bitMask` is `null`:
```suggestion
    return bitMask == null
        ? 1 << bitPos
        : bitMask.intValue() | 1 << bitPos;
```